blueprints-provider/python/RELEASE.md

```diff
@@ -0,0 +1,4 @@
+## Release matrix
+
+| Docker Image Tag | Python Library Version |
+|------------------|------------------------|
```

Comment:
This is probably needed for package release.
If the docker image does not exist, use the 0.0.1 tag.